### DISCUSSION TITLE
Copying contents from the fields of the QSO window

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#214] [Q&A]
Hi,

did you change anything in the input / copy method in the fields of the QSO window?

I observe the following effect:

![Screenshot_20230613_074552](https://github.com/foldynl/QLog/assets/26311402/28b352a5-ea76-4d92-b920-1b3e37b0c95f)

When I double-click and highlight a callsign in the Call field, the background turns blue. 

![Screenshot_20230613_074616](https://github.com/foldynl/QLog/assets/26311402/ef17f632-1f15-4131-b6a6-6309d2a27368)

If I right-click on the selected callsign to copy it, the context menu with "Coppy" appears, but at the same time the selection of the callsign is canceled, so that with this method the call cannot be copied to the clipboard. 

![Screenshot_20230613_074846](https://github.com/foldynl/QLog/assets/26311402/57843c07-9117-4f2d-ab3b-555fdc2a124e)

"Ctrl-C" in a marked field content works.

In the fields "Name" or "QTH", by example, there is the same behavior.

**Comment by foldynl** [comment]
Yes, I did. This release introduced a feature that a long text is shown from the beginning when it is not focused. Unfortunately, It seems that it has a negative side-effect. I am opening an issue for it. It must be fixed or reverted.

**Comment by dl2ki** [comment]
I don't think you have to undo it, because it's a good improvement. Copy & Past is a standard feature and in QT-Linguist it works as well. It will probably be, as so often, due to some small thing.

**Comment by foldynl** [comment]
Yeah, you are right. It was a small thing. I do not revert the feature, just fixing. Fix  is prepared (#215).
